src/services/completions.ts

```diff
@@ -1144,9 +1144,14 @@ namespace ts.Completions {
             }
 
             if (isMetaProperty(node) && (node.keywordToken === SyntaxKind.NewKeyword || node.keywordToken === SyntaxKind.ImportKeyword)) {
-                const completion = (node.keywordToken === SyntaxKind.NewKeyword) ? "target" : "meta";
-                symbols.push(typeChecker.createSymbol(SymbolFlags.Property, escapeLeadingUnderscores(completion)));
-                return;
+                const metaPropertyAccessIdentifier = node.name.text;
+                const dotToken = node.getChildAt(1);
+                // Check whether this MetaProperty node already had its metadata accessed
+                if (dotToken === contextToken || metaPropertyAccessIdentifier !== "target" && metaPropertyAccessIdentifier !== "meta") {

```

Comment:
I don't think you need any of the text validation.. 
Just adding `contextToken === contextToken` should be sufficient. Also please include this by changing outer if statement as:
```ts
            if (isMetaProperty(node) &&
                (node.keywordToken === SyntaxKind.NewKeyword || node.keywordToken === SyntaxKind.ImportKeyword) &&
                contextToken === node.getChildAt(1)) {
```




```